acceptance/2016nano/inspec/verify.rb

```diff
@@ -0,0 +1,3 @@
+describe file("c:/opscode/angry-omnibus-toolchain") do
+  it { should be_directory }
+end
```

Comment:
Can we add some checks that use [InSpec's command resource](http://inspec.io/docs/reference/resources/command/) to ensure a few of the toolchain's actual tools are installed and are functional. Maybe check for `git`, `ruby` and `gcc`? 

If the`PATH` is configured per:
https://github.com/chef-cookbooks/omnibus/blob/master/recipes/_environment.rb#L24-L28

You can easily just call `--version` on each of those tools...like:
https://github.com/chef-cookbooks/omnibus/blob/master/recipes/_environment.rb#L57-L61